### DISCUSSION TITLE
Can specify multiple values to the nfs server?

### DISCUSSION THREAD

**Discussion by zhucan** [ceph/ceph-csi#3295] [Q&A]
https://github.com/ceph/ceph-csi/blob/1bd6297ecbdf11f1ebe6a4b20f8963b4bcebe13b/examples/nfs/storageclass.yaml#L13

Deploy the ceph with rook and set the mode of network is "hostNetwork: true".

**Comment by nixpanic** [comment]
You would set this to a service that points to the NFS-Ganesha pods that Rook deployed. No need for specifying multiple hostnames that way.

**Comment by zhucan** [comment]
Cound you print the example yaml? @nixpanic
